pages/services/kafka/2.6.0-2.2.1/notes/index.md

```diff
@@ -0,0 +1,151 @@
+---
+layout: layout.pug
+navigationTitle:
+excerpt: Release notes for 2.6.0-2.2.1
```

Comment:
Can we rename the parent directory to `release-notes`, similar to the other FWs?

Reply:
Done